olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -382,6 +382,32 @@ class DATASERVICE_READ_API VersionedLayerClient final {
    */
   bool IsCached(const geo::TileKey& tile) const;
 
+  /**
+   * @brief Protects tile keys from eviction.
+   *
```

Comment:
Please add more description. What does it mean if you call this with a TileKey: will be removed from LRU, list of protected keys will be stored into cache, etc.